acceptance-tests/domain_fronting_test.go

```diff
@@ -171,6 +153,9 @@ var _ = Describe("Domain fronting", func() {
 			expect421(nonMTLSClient.Do(req))
 			expect421(mtlsClient.Do(req))
 
+			By("Sending a request to HAProxy with a mismatched SNI and Host header it returns a 400")
```

Comment:
Shouldn't this say rather: "Sending a request to HAproxy with no Host header it returns a 400"?
SNI check happens after the 400 so it should be unrelated.

Reply:
fixed